db/migrate/20180322104809_add_index_dimensions_items_locale.rb

```diff
@@ -0,0 +1,6 @@
+class AddIndexDimensionsItemsLocale < ActiveRecord::Migration[5.1]
```

Comment:
I am not sure we need this index, as we never search only by locale.

Reply:
Do we need to add it to the index that includes content_id and latest?

Reply:
Not sure Pete. We could postpone the decision?